bin/install.sh

```diff
@@ -3,4 +3,5 @@
 (
 cd "$( dirname "${BASH_SOURCE[0]}" )/.."
 pip install -r requirements.txt
+pip install -r dev.requirements.txt
```

Comment:
Do we want a `-U` in these pip calls in case the requirements files are updated?